src/runtime/virtcontainers/qemu.go

```diff
@@ -689,6 +691,25 @@ func (q *qemu) CreateVM(ctx context.Context, id string, network Network, hypervi
 	return err
 }
 
+func (q *qemu) checkBpfEnabled() {
+	if q.config.SeccompSandbox != "" {
+		out, err := pkgUtils.RunCommand([]string{"cat", "/proc/sys/net/core/bpf_jit_enable"})
```

Comment:
@fengwang666 This works, but this does spawn a separate process for reading the bpf_jit_enable file. I think it would be better to just read the file contents without the overhead of running a separate process to do so.

Reply:
@amshinde Following on your other comment, if this flag needs to be enabled inside the guest, then the check logic needs to be in the agent? But I don't understand why this flag needs to be enabled inside the guest instead of host OS. Can you explain a bit more?

Reply:
@fengwang666 Provided clarification below.
But for the above comment, I dont think you need to exec into a separate process.

Reply:
I would also think that reading the file directly would be cleaner here.

Reply:
agreed. updated. @amshinde @pmores